papers.html

```diff
@@ -159,24 +159,66 @@ <h5>Kruckow et al. 2024</h5>
+    </div>
+  </div>
+  <hr/>
+</div>
+
+<div class="container p-4">
+  <h1><a href="https://arxiv.org/abs/2403.17279" target="_blank" style="color: rgb(0,0,0)">Stellar Spin Down in Post-Mass Transfer Binary Systems</a></h1>
+  <p></p>
+  <h5>Sun et al. 2024, accepted in ApJ</h5>
+  <p></p>
+
+  <p>
+    <button class="btn btn-secondary" type="button" data-toggle="collapse" data-target="#Sun20240807" aria-expanded="false" aria-controls="Sun20240807">
+      Details
+    </button>
+  </p>
+    <div class="collapse" id="Sun20240807">
+      <div class="card card-body">
+        <p><b>Authors: </b> Meng Sun, Seth Gossage, Emily M. Leiner, Aaron M. Geller</p>
+        <p><b>Access: </b> <a href="https://arxiv.org/abs/2403.17279" target="_blank">arXiv:2403.17279</a> || <a href="https://ui.adsabs.harvard.edu/abs/2024ApJ...971...80S/abstract" target="_blank">NASA ADS</a></p>
+        <p><b>Abstract: </b>Motivated by measurements of the rotation speed of accretor stars in post-mass-transfer (post-MT) systems, we investigate how magnetic braking affects the spin-down of individual stars during binary evolution with the \texttt{MESAbinary} module. Unlike the conventional assumption of tidal synchronization coupled with magnetic braking in binaries, we first calculate whether tides are strong enough to synchronize the orbit. Subsequently, this influences the spin-down of stars and the orbital separation. In this study, we apply four magnetic braking prescriptions to reduce the spin angular momentum of the two stars throughout the entire binary evolution simulation. Our findings reveal that despite magnetic braking causing continuous spin-down of the accretor, when the donor begins to transfer material onto the accretor, the accretor can rapidly spin up to its critical rotation rate. After MT, magnetic braking becomes more important in affecting the angular momentum evolution of the stars. Post-MT accretor stars thus serve as a valuable testbed for observing how the magnetic braking prescriptions operate in spinning down stars from their critical rotation, including the saturation regimes of the magnetic braking. The rotation rate of the accretor star, combined with its mass, could provide age information since the cessation of MT. By comparing the models against observation, the magnetic braking prescription by \citet{2018ApJ...862...90G} is found to better align with the rotation data of post-MT accretors. </p>
```

Comment:
Please convert the LaTeX to plain text.

Reply:
```suggestion
        <p><b>Abstract: </b>Motivated by measurements of the rotation speed of accretor stars in post-mass-transfer (post-MT) systems, we investigate how magnetic braking affects the spin-down of individual stars during binary evolution with the MESAbinary module. Unlike the conventional assumption of tidal synchronization coupled with magnetic braking in binaries, we first calculate whether tides are strong enough to synchronize the orbit. Subsequently, this influences the spin-down of stars and the orbital separation. In this study, we apply four magnetic braking prescriptions to reduce the spin angular momentum of the two stars throughout the entire binary evolution simulation. Our findings reveal that despite magnetic braking causing continuous spin-down of the accretor, when the donor begins to transfer material onto the accretor, the accretor can rapidly spin up to its critical rotation rate. After MT, magnetic braking becomes more important in affecting the angular momentum evolution of the stars. Post-MT accretor stars thus serve as a valuable testbed for observing how the magnetic braking prescriptions operate in spinning down stars from their critical rotation, including the saturation regimes of the magnetic braking. The rotation rate of the accretor star, combined with its mass, could provide age information since the cessation of MT. By comparing the models against observation, the magnetic braking prescription by <a href="http://doi.org/10.3847/1538-4357/aace5d">Garraffo et al. (2018b)</a> is found to better align with the rotation data of post-MT accretors. </p>
```

Reply:
thank you, updated